app/shared/directives/ipaTermSelectorDropdown/ipaTermSelectorDropdown.html

```diff
@@ -1,15 +1,13 @@
 <div class="ipa-term-selector-dropdown header--work-group btn-group" role="group" aria-label="...">
-
-	<button ng-if="readOnlyMode == true"
-					class="ipa-term-selector-dropdown__button btn btn-default dropdown-toggle"
-					type="button"
-					aria-haspopup="true" aria-expanded="true">
-		<span>
-			{{ currentTermDescription }}
-		</span>
-	</button>
-
-	<button ng-if="readOnlyMode == false"
+	<div ng-if="readOnlyMode == true"	
```

Comment:
Why switch from `button` to `div`?

Reply:
Because if it is a button, you still have interaction when hover. I think if you don't have anything to click for there is no point to have a button

![2018-12-18_15-44-59](https://user-images.githubusercontent.com/13563329/50189954-3ebac900-02dc-11e9-8108-5840fd9f7577.png)